web/js/containers/toolbar.js

```diff
@@ -181,14 +181,26 @@ class toolbarContainer extends Component {
   }
 
   renderShareButton() {
-    const { faSize, openModal, isDistractionFreeModeActive } = this.props;
+    const {
+      faSize,
+      openModal,
+      isDistractionFreeModeActive,
+      isMobile,
+    } = this.props;
     const buttonId = 'wv-share-button';
     const labelText = 'Share this map';
+    const mobileWvToolbarButtonStyle = {
+      fontSize: '14.3px',
+      height: '44px',
+      margin: '0 0 0 4px',
+      padding: '5.72px 9.1px',
+    };
     return !isDistractionFreeModeActive && (
       <Button
         id={buttonId}
         className="wv-toolbar-button"
         aria-label={labelText}
+        style={isMobile ? mobileWvToolbarButtonStyle : null}
```

Comment:
Sorry missed this one yesterday. I agree that putting the conditional in the variables instead of the JSX is cleaner. I've updated the these instances.